spec/factories/items.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :item do
+    name "MyString"
```

Comment:
I have to go back and read on FactoryGirl, I didn't input this spec, and I'm not sure I know how to use it properly.  I won't have time during the day Friday to go over this, so we may talk about it during our call, if we have time.